### PR TITLE
Added package for cyrus-sasl

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
This is a requirement for building a conda forge variant of `sasl` which is a dependency of `thrift_sasl`.  This is part of the work to enable OSx builds for this library

**Comment by mariusvniekerk** [comment]
@wesm @martindurant Are you okay with being added as maintainers here?

**Comment by martindurant** [comment]
I am happy to help if I can. I don't know a whole lot about cyrus, though

**Comment by wesm** [comment]
OK to add me as a maintainer

**Comment by ericdill** [comment]
Thanks @mariusvniekerk

File: cyrus-sasl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: ftp://ftp.cyrusimap.org/cyrus-sasl/{{ name }}-{{ version }}.tar.gz
+  {{ hashtype }}: {{ hash }}
+
+build:
+  number: 1
```
Comment (ericdill) [inline cyrus-sasl/meta.yaml]: Can you note why this is build number 1 after the feedstock is created?

**Comment by jakirkham** [comment]
Also appears this recipe wasn't in the `recipes` directory. So it should be moved into the `recipes` directory so it can be converted. Would be a good time to correct the build number as well.

**Comment by jakirkham** [comment]
Friendly nudge @mariusvniekerk. 😉

**Pull request merged**